keps/sig-scheduling/2779-support-custom-queue-for-the-scheduler/README.md

```diff
@@ -396,3 +369,11 @@ Why should this KEP _not_ be implemented?
 An alternative is to enhance the current queue to meet some requests. The drawback
 is that we cannot handle all the potential requests as the effort is big, we should
 give the ball to the users, and they can do what they want.
+
+Another alternative is to suspend a job and don't let it go unless the resource
+sharing policy will not be broken. The drawback is that all the pods in the job will
+be impacted, it cannot handle the case that only some pods need to be touched.
+
+Another alternative is to make the pod that will break the resource sharing policy with
+`Unschedulable`. The drawback is that making such a decision needs the status of all other
+pods, the logic is complex, and the performance will be impacted.
```

Comment:
In the proposal, don't you have to know the status of all pods during sorting? It looks like the latency is moving from one place to another.

Reply:
Oh, the `status` here is not clear. I update it to `The drawback is that making such a decision needs to know whether other
pods can run or not, ......`. To mark a pod as `Unschedulable`, the alternative needs to predict whether the competitor pods can run or not, this is complex.

In my proposal, the pods for different users will be put into different queues, and once the scheduler want to pop a pod, it will select the most under-used queue and pop one, the scheduler needn't care about other pods in other queues, logic is simple, there is no latency.

Reply:
Can't you keep the same information about usage ratios in the PreFilter plugin? The latency should be similar.